src/programs/ParseJuliaPrograms.jl

```diff
@@ -109,15 +109,15 @@ end
 """ Make a lookup table assigning names to generators or term constructors.
 """
 function make_lookup_table(pres::Presentation, syntax_module::Module, names)
-  theory = GATs.theory(syntax_module.theory())
-  terms = Set([ term.name for term in theory.terms ])
+  theory = syntax_module.THEORY_MODULE.Meta.theory
```

Comment:
I wonder whether there should be an accessor function to get the GAT associated with the syntax module, not unlike what we had before, so that we don't have to expose the field `THEORY_MODULE` in all its ugly CAPS.

Reply:
Sounds reasonable, we could have a function `syntax_module.theory()` exactly like before.

Reply:
Here's the question though: should `syntax_module.theory()` return `syntax_module.THEORY_MODULE.Meta.theory`, `syntax_module.THEORY_MODULE` or `syntax_module.THEORY_MODULE.Meta.T`? The closest analogy to Catlab would be `syntax_module.THEORY_MODULE.Meta.T`, but that also seems like the least useful.